spec/requests/api/v1/docs/articles_spec.rb

```diff
@@ -8,20 +8,86 @@
 RSpec.describe "Api::V1::Docs::Articles" do
   let(:organization) { create(:organization) } # not used by every spec but lower times overall
   let(:tag) { create(:tag, :with_colors, name: "discuss") }
-  let(:article) { create(:article, featured: true, tags: "discuss", published: true) }
+  let(:published_article) { create(:article, featured: true, tags: "discuss", published: true) }
   let(:unpublished_aricle) { create(:article, published: false) }
   let(:api_secret) { create(:api_secret) }
   let(:user) { api_secret.user }
+  let(:user_article) { create(:article, featured: true, tags: "discuss", published: true, user_id: user.id) }
   let(:Accept) { "application/vnd.forem.api-v1+json" }
+  # let!(:sample_articles) { create_list(:article, 3) }
 
   before { stub_const("FlareTag::FLARE_TAG_IDS_HASH", { "discuss" => tag.id }) }
 
   describe "GET /articles" do
     before do
-      article.update_columns(organization_id: organization.id)
+      published_article.update_columns(organization_id: organization.id)
     end
 
     path "/api/articles" do
+      post "Publish article" do
+        tags "articles"
+        description "This endpoint allows the client to create a new article.
+
+\"Articles\" are all the posts that users create on DEV that typically show up in the feed. They can be a blog post, a discussion question, a help thread etc. but is referred to as article within the code."
+        operationId "createArticle"
+        produces "application/json"
+        consumes "application/json"
+        parameter name: :article, in: :body, schema: {
```

Comment:
Should we move this schema to the swagger_helper since we duplicate it on line 244?

Reply:
Great suggestion, thanks! Just included it 👍🏼